charts/linkerd-crds/values.yaml

```diff
@@ -1 +1,2 @@
+enableGrpcRoutes: true
```

Comment:
Note that this isn't used in the manifest. I think it would be fine to to gate all of it under the single enableHttpRoutes for now. If you disable enableHttpRoutes, I don't think you mean to leave enableGrpcRoutes on.

Reply:
Drat! Must not've been paying enough attention to that tab-completion.

I think gating all the gateway crds under the same flag make sense. I'd advocate to change the flag name though, for clarity's sake. Maybe `enableGatewayRoutes` or `enableGatewayCRDs`?

Reply:
I'm not opposed to having a more generic flag, but we already have the existing flag out there and in docs. I'd probably default to doing the smallest change for now and flag this as an area for potential followup

Reply:
@olix0r reverted key name to `enableHttpRoutes`